carrotServer/src/main/java/cds/carrot/org/carrotServer/controller/employer/EmployerController.java

```diff
@@ -0,0 +1,32 @@
+package cds.carrot.org.carrotServer.controller.employer;
+
+import cds.carrot.org.carrotServer.common.dto.ErrorType;
+import cds.carrot.org.carrotServer.common.dto.JsonResponse;
+import cds.carrot.org.carrotServer.common.dto.SuccessType;
+import cds.carrot.org.carrotServer.controller.employer.dto.response.EmployerResponseDto;
+import cds.carrot.org.carrotServer.exception.BadRequestException;
+import cds.carrot.org.carrotServer.service.employer.EmployerService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/employer")
+public class EmployerController {
+
+    private final EmployerService employerService;
+
+    @GetMapping
```

Comment:
@ GetMapping 뒤에 어떤 path 에 대한 정보가 없는데,
@ PathVariable Long userId 이렇게 두면, 어떤 값으로 바인딩되지 몰라서 정상적으로 동작하지 않겠네요~
```suggestion
    @GetMapping("/{userId}")
```

Reply:
음 하나만 있을때는 자동으로 바인딩이되나.. ㅋㅋ 이건 확실하지 않네요
근데 저렇게 명시해주는게 좋아요

Reply:
`@GetMapping("/{userId}/reviewrs")` 이렇게 적으려고합니다요